_pytest/python.py

```diff
@@ -711,31 +711,26 @@ class Metafunc(fixtures.FuncargnamesCompatAttr):
     They help to inspect a test function and to generate tests according to
     test configuration or values specified in the class or module where a
     test function is defined.
-
-    :ivar fixturenames: set of fixture names required by the test function
```

Comment:
I had to change from docstring to inline comments because of sphinx-doc/sphinx#2549. :disappointed: